README.md

```diff
@@ -119,14 +123,15 @@ public class StringExample {
 
         // The SDK may add information to the encryption context, so check to
         // ensure all of the values are present
-        for (final Map.Entry<String, String> e : context.entrySet()) {
-            if (!e.getValue().equals(decryptResult.getEncryptionContext().get(e.getKey()))) {
+        if (!context.entrySet().stream
+            .allMatch( e -> e.getValue().equals(decryptResult.getEncryptionContext().get(e.getKey())))) {
                 throw new IllegalStateException("Wrong Encryption Context!");
-            }
         }
 
+        assert Arrays.equals(decryptResult.getResult(), data.getBytes(StandardCharsets.UTF_8));
+
         // The data is correct, so output it.
-        System.out.println("Decrypted: " + decryptResult.getResult());
+        System.out.println("Decrypted: " + Arrays.toString(decryptResult.getResult()));
```

Comment:
We should print the decrypted data as a string since that's how it arrived:

```suggestion
        System.out.println("Decrypted: " + new String(decryptResult.getResult(), StandardCharsets.UTF8));
```